[IMP] Contacts: partner webpage edit (#9821)
odoo/documentation: content/applications/essentials/contacts.rst

Next is the :guilabel:`Partner Assignment` tab, which by default, includes a :guilabel:`Geolocation`
section, and other partner options, including :guilabel:`Partner Activation` and :guilabel:`Partner
Review` configurations. These are **only** present when the *Resellers* module is installed.

> With the Odoo *Website* and *Resellers* apps installed, a new webpage is created to display a list
> of all active partners from the *CRM* app.
> 

Reviewer: ```suggestion
```

> To add partners to this webpage, return to :menuselection:`CRM app --> Sales --> Customers`, and
> click the Kanban card for a partner. From that partner's contact form, click the :guilabel:`Go to
> Website` smart button at the top of the page to open that partner's webpage.
> 
> Next, click :guilabel:`Edit` at the top-right of the partner's webpage, and use the :doc:`building
> blocks <../websites/website/web_design/building_blocks>` to add any additional design elements, or

Reviewer: ```suggestion
```

> 
> Repeat these steps for all partners.
> 

Reviewer: ```suggestion
```

> .. seealso::
>    :doc:`Resellers <../sales/crm/track_leads/resellers>`
> 

Reviewer: Instead of re-explaining publishing partners (in the paragraphs above the seealso), just link to the new _Resellers_ documentation
```suggestion
.. seealso::
   Follow the :doc:`Resellers documentation <../sales/crm/track_leads/resellers>` for more
   information on publishing partners on the website.
   
```

when both fields are filled in.

Alternatively, to offer a free membership, tick the :guilabel:`Free Member` checkbox, in the
:guilabel:`Membership` tab of a contact form.

Publish members directory
~~~~~~~~~~~~~~~~~~~~~~~~~

> To publish a list of active members on the website, the *Online Members Directory* application must
> first be installed. Navigate to the :menuselection:`Apps application` and remove the
> :guilabel:`Apps` filter from the :guilabel:`Search...` bar. Then, search for `Online Members
> Directory`.
> 
> Click :guilabel:`Install` on :guilabel:`Online Members Directory` card that appears. Doing so
> installs the module and returns to the main Odoo dashboard.

Reviewer: I dont think it is necessary to explain how to install the module, can just reference the existing documentation.

Then make sure any referenced documentation is contextual, rather than the vague "these steps" :)

Also, detail the `/members` page URL.
![members-menu-item](https://github.com/odoo/documentation/assets/36018073/75749247-d609-4535-8df7-b8e37a74f884)

```suggestion
To publish a list of active members on the website, the *Online Members Directory* application must
first be :ref:`installed <general/install>`. After installing the module, add the `/members` page to
the website's menu by :doc:`editing the website menu <../websites/website/pages/menus>`.
```

> After installing the module, follow :ref:`these steps <website/regular-menus>` to add the members
> directory to the website menu.
> 

Reviewer: ```suggestion
```
